docs/spec_draft.md

```diff
@@ -1377,6 +1378,42 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.log_plus_one
+
+### Semantics
+
+Computes element-wise logarithm of `operand` tensor plus one and produces a
+`result` tensor. More formally,
+`result[i0, ..., iR-1] = log(operand[i0, ..., iR-1] + 1)`, where `log` is the
```

Comment:
`log1p(x)` is different from `log(x + 1)` in that it's more accurate for small `x`. You can read more about this e.g. in https://numpy.org/doc/stable/reference/generated/numpy.log1p.html.

As a result, the formalization will need to be framed in the terms of the `log1p` operation from IEEE-754. Also, I think it would be preferable to phrase the first sentence similarly to #502, i.e. "Performs element-wise logarithm of one plus operation", emphasizing that this is a separate operation from a combination of a logarithm and plus.